pubspec.yaml

```diff
@@ -23,6 +23,7 @@ dependencies:
   file_picker: ^4.1.6
 
 dev_dependencies:
+  flutter_lints: ^1.0.4
```

Comment:
Thanks for adding this and all the lint fixes!

Reply:
I just made a separate PR for this so it can be merged now. https://github.com/anthem-audio/anthem/pull/12